RELEASE.md

```diff
@@ -11,6 +11,9 @@
 # Upcoming Release 0.18.9
 
 ## Major features and improvements
+* `OmegaConfigLoader` will return a `dict` instead of `DictConfig`.
+* `kedro run --params` now update interpolated parameters correctly.
```

Comment:
That's also just for the `OmegaConfigLoader` right?

Reply:
That's correct, I will make changes to this

Reply:
I change `templated_param` -> `interpolated_param` to align with the OmegaConf terminology.

I have changed the `global` name, but I think we need to come up with a better name since we are calling it global with `OmegaConfigLoader`, i.e.`parameters_global.yml` is what suggested in the doc.